src/Controller/AssetsController.php

```diff
@@ -24,6 +25,18 @@ class AssetsController extends Controller
     private const WIDTH = 250;
     private const HEIGHT = 170;
 
+    /**
+     * @Route("/assets_files/{path}", requirements={"path": ".+"}, name="asset_files_url")
+     * @Method("GET")
+     * @Cache(maxage=900, smaxage=900)
+     */
+    public function assetFilesAction(FilesystemInterface $filesystem, string $path)
+    {
+        return new Response($filesystem->read($path), Response::HTTP_OK, [
```

Comment:
this is very dangerous.

what if i query this route with path `../../app/config.yml` for example ?

Reply:
How can I fix it then ? I can change it to have file id in db and then use the path in db ?

Reply:
using the ID would still be a better idea yes, since you can throw a 404 if the file does not exist or is not meant to be exposed.

can't Nginx do the job ? what is the purpose of this route except serving a file from a path ?
ping @B-Galati

Reply:
I missed that comment sorry @Remg 
We would need to understand the goal of this action 1st.